application/predictionengine/src/predictionengine/long_short_term_memory.py

```
@@ -58,5 +58,6 @@ def forward(
 
             outputs.append(hidden_state[-1])
 
-        output_tensor = Tensor.stack(*outputs, dim=1)
+        # TODO: add check for empty outputs or size equal to one
```

Pylint (3.3.7)

[warning] TODO: add check for empty outputs or size equal to one

(W0511)